core/templates/services/user-backend-api.service.ts

```diff
@@ -57,16 +57,17 @@ export interface UserContributionRightsDataBackendDict {
   providedIn: 'root'
 })
 export class UserBackendApiService {
-  constructor(private http: HttpClient) {}
+  constructor(private httpClient: HttpClient) {}
```

Comment:
why this change? (I think we have the old pattern everywhere in the codebase)

Reply:
Oh! I have been doing this in all of my PRs. Following the convention of the first letter lowercase for the object name.
Looking deeply into the codebase, looks like we are going with `http` in most places. Reverting this change.

Reply:
done.